### PR TITLE
Developer guidelines

### LINKED ISSUE #53: Developer guidelines

### PR DESCRIPTION
**Pull request by cforgaci**
Closes #53

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] Yes
+- [ ] No, and this is why: _please replace this line with details on why tests
+      have not been included_
+- [ ] I need help with writing tests
+
+## Added entry in changelog?
+_For user-facing changes, add a line describing the changes in NEWS.md_
+
+- [ ] Yes
+![alt_text](gif_link)
```
Comment (fnattino) [inline .github/PULL_REQUEST_TEMPLATE.md:41]: Is this something that is here by mistake?
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Added/updated tests?
+_We encourage you to keep the code coverage percentage at 75% and above._
+
+- [ ] Yes
+- [ ] No, and this is why: _please replace this line with details on why tests
+      have not been included_
+- [ ] I need help with writing tests
+
+## Added entry in changelog?
+_For user-facing changes, add a line describing the changes in NEWS.md_
```
Comment (fnattino) [inline .github/PULL_REQUEST_TEMPLATE.md:38]:
```suggestion
_For user-facing changes, add a line describing the changes in [NEWS.md](/NEWS.md)_
```

**Review by fnattino — CHANGES_REQUESTED** [review]
Great, nice templates, I have left two minor comments! I think things that are left to do here are:
- [ ] add NEWS.md
- [ ] add contributing guidelines (CONTRIBUTING.md)
- [ ] Either add links to code-of-conduct, contributing, changelong either in the README or in a separate file (developers_guide.md?). In the same document, describe the release procedure.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (cforgaci) [reply .github/PULL_REQUEST_TEMPLATE.md:41]: Yes, that's a mistake. Thanks!

**Comment by cforgaci** [comment]
Thanks! I will take care of points 1 and 3. Note that there was a CONTRIBUTING.md already, but it is not visible in this PR as I did not change anything in it.

**Comment by fnattino** [comment]
>  Note that there was a CONTRIBUTING.md already, but it is not visible in this PR as I did not change anything in it.

Is it possible that it is not checked in? I don't seem to find it in the repository

**Comment by cforgaci** [comment]
> > Note that there was a CONTRIBUTING.md already, but it is not visible in this PR as I did not change anything in it.
> 
> Is it possible that it is not checked in? I don't seem to find it in the repository
I can see it in .github/CONTRIBUTING.md

**Comment by cforgaci** [comment]
> I will take care of points 1 and 3.

These are done too.

**Comment by fnattino** [comment]
> I can see it in .github/CONTRIBUTING.md. @fnattino let me know when you found it.

Sorry, my bad, haven't looked in that folder, but it makes sense it is there!

**Review by fnattino — APPROVED** [review]
Looks great, thanks!

**Pull request merged**